src/Datadog.Trace.ClrProfiler.Native/cor_profiler.cpp

```diff
@@ -403,17 +403,27 @@ HRESULT STDMETHODCALLTYPE CorProfiler::JITCompilationStarted(
         continue;
       }
 
-      if (method_replacement.wrapper_method.method_signature.data.size() < 3) {
+      auto wrapper_method_signature_size =
+          method_replacement.wrapper_method.method_signature.data.size();
+
+      if (wrapper_method_signature_size < 3) {
         // This is invalid, we should always have the wrapper fully defined
         // Minimum: 0:{CallingConvention}|1:{ParamCount}|2:{ReturnType}
         // Drop out for safety
+        if (debug_logging_enabled) {
+          Debug("JITCompilationStarted skipping method: function_id=",
```

Comment:
Would it be amenable to add type name and method name to this?

Reply:
Added method tokens and names in f9b6c2ab2748f0ae1e4f579157a455d9ffb71333.